netbox/extras/views.py

```diff
@@ -753,6 +753,8 @@ def get(self, request):
 
         for _scripts in scripts.values():
             for script in _scripts.values():
+                # Prevent django from instantiating the class on all accesses
+                script.do_not_call_in_templates = True
```

Comment:
Seems like this should be set on the base class, rather than being set by the view.